flytepropeller/pkg/controller/nodes/array/handler.go

```diff
@@ -631,6 +655,47 @@ func (a *arrayNodeHandler) Handle(ctx context.Context, nCtx interfaces.NodeExecu
 	return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoRunning(&handler.ExecutionInfo{})), nil
 }
 
+// OffloadLargeLiteral offloads the large literal if meets the threshold conditions
+func (a *arrayNodeHandler) OffloadLargeLiteral(ctx context.Context, datastore *storage.DataStore, dataReference storage.DataReference, toBeOffloaded *idlcore.Literal) error {
+	// TODO : also add sdk version checks to ensure that the literal is not offloaded if the sdk version does not support it
```

Comment:
this is now done in this change, right? :)

Reply:
thanks. stale comment